pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -2522,6 +2522,9 @@ type S3Bucket struct {
 
 	// CASecret specifies the secret that contains the CA, used to verify the TLS connection
 	CASecret *apiv1.SecretKeySelector `json:"caSecret,omitempty" protobuf:"bytes,11,opt,name=caSecret"`
+
+	// SessionTokenSecret is used for credentials which are ephemeral (e.g. IAM assume role or S3 access grant)
```

Comment:
```suggestion
	// SessionTokenSecret is used for ephemeral credentials like an IAM assume role or S3 access grant
```

- simplify a bit, per k8s style guide ("credentials which are ephemeral" -> "ephemeral credentials")
- avoid Latin phrases, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases)

Reply:
this suggestion was missed

Reply:
Sorry, fixed!